### ISSUE TITLE
NotImplementedError: Multi-part geometries do not themselves provide the array interface

### ISSUE THREAD

**Issue by epierson9** [gboeing/osmnx#326]
When using footprints_from_place, I get a NotImplementedError for certain queries. In particular, running

```
>>> import osmnx as ox
>>> ox.footprints.footprints_from_place(place='Alabama', footprint_type='amenity')
```

results in

```
NotImplementedError: Multi-part geometries do not themselves provide the array interface
```

I am using osmnx version 0.11dev, installed from source using `python setup.py install` in a conda environment. Python version is 3.7.2.

**Comment by gboeing** [comment]
should be resolved by #327 now

**Comment by epierson9** [comment]
Thanks very much -- confirming this fixed it. This is a very elegant package which will be extremely useful for our project -- thanks for all the work building and maintaining it!
